wp-content/themes/core/assets/pcss/navigation/enhanced-nav-menu.pcss

```diff
@@ -0,0 +1,22 @@
+/* -------------------------------------------------------------------------
+ *
+ * Pattern: Enhanced Navigation Menu
+ *
+ * minimal styles for enhanced navigation layout.
+ * ------------------------------------------------------------------------- */
+
+.nav-menu__enhanced {
+	display: grid;
+	grid-gap: var(--grid-gutter);
+	grid-template-columns: 1fr;
+
+	@container footer (min-width: 980px) {
+		display: grid;
+		grid-template-columns: minmax(auto, 400px) 1fr;
+		align-items: start;
+	}
+
+	.wp-block-navigation:not(ul) {
```

Comment:
This seems like a very specific condition. Can you elaborate on it in a comment?

Reply:
Nav block does not support margin or padding (i know, crazy), only blockGap. this was added for admin editor view only. Not ideal to put it in the theme side of the CSS but we currently do not have an parts/patterns editor and theme stylesheets separated.

Reply:
Found a better way to fix this in theme.json